pa11ycrawler/pipelines.py

```diff
@@ -108,6 +109,44 @@ def write_pa11y_config(self, item):
         config_file.close()
         return config_file
 
+    def check_title_match(self, expected_title, pa11y_output, logger):
+        """
+        Check if Scrapy reports any issue with the HTML <title> element.
+        If so, compare that <title> element to the title that we got in the
+        A11yItem. If they don't match, something is screwy, and pa11y isn't
+        parsing the page that we expect.
+        """
+        if not pa11y_output:
+            # no output from pa11y, nothing to check.
+            return
+        pa11y_results = json.loads(pa11y_output).get("results")
+        if not pa11y_results:
+            return
+        title_errs = [err for err in pa11y_results
+                      if err["html"].startswith("<title")]
+        for err in title_errs:
+            title_elmt = html.fragment_fromstring(err["html"])
+            # pa11ycrawler will elide the title, so grab whatever true
```

Comment:
no i was able to pick up what it meant in the subsequent lines of code. the word choice is fine IMO. TIL man.